src/geometry.F90

```diff
@@ -189,27 +187,28 @@ recursive subroutine find_cell(p, found, search_cells)
         ! CELL CONTAINS LOWER UNIVERSE, RECURSIVELY FIND CELL
 
         ! Create new level of coordinates
```

Comment:
This comment should now be something like "Index lower level ..." instead of "Create new level ..."